<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QWidget" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>521</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="log_horizontalLayout">
      <item>
       <widget class="QLabel" name="log_label">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="log_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="pass_horizontalLayout">
      <item>
       <widget class="QLabel" name="pass_label">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pass_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mail_horizontalLayout">
      <item>
       <widget class="QLabel" name="mail_label">
        <property name="text">
         <string>Почта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mail_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="teacher_but">
      <property name="text">
       <string>Я учитель</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="but_horizontalLayout">
      <item>
       <widget class="QPushButton" name="but_sign_in">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="but_sign_up">
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="but_exit">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
